Modules/OptionHolder.cs

```diff
@@ -2670,6 +2677,11 @@ public static void Load()
             .SetValueFormat(OptionFormat.Times);
         AutoKickStartAsBan = BooleanOptionItem.Create(19312, "AutoKickStartAsBan", false, TabGroup.SystemSettings, false)
             .SetParent(AutoKickStart);
+
+        TempBanPlayersWhoKeepQuitting = BooleanOptionItem.Create(44440, "TempBanPlayersWhoKeepQuitting", false, TabGroup.SystemSettings, false);
+        QuitTimesTillTempBan = IntegerOptionItem.Create(44441, "QuitTimesTillTempBan", new(1, 15, 1), 4, TabGroup.SystemSettings, false)
+            .SetParent(TempBanPlayersWhoKeepQuitting)
+            .SetValueFormat(OptionFormat.Times);
```

Comment:
This setting is not used anywhere, lol

Reply:
Huh?
(Main.PlayerQuitTimes[data?.FriendCode] >= Options.QuitTimesTillTempBan.GetInt())

Reply:
Damn, there are 2 settings and not 1, lol

But `TempBanPlayersWhoKeepQuitting` not using

Reply:
Or wait, if `TempBanPlayersWhoKeepQuitting` is false, `QuitTimesTillTempBan` will be work?